src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -1215,6 +1216,7 @@ class DevTask extends AbstractFeatureTask {
 
     @TaskAction
     void action() {
+        HashSet<Project> allProjects = new HashSet<>()
```

Comment:
Shouldn't the variable type just be `Set<Project>`? And why do you not include `Project` when you create the `HashSet`?

Reply:
Changed to Set